internal/test/testdata/verify/suite.yaml

```diff
@@ -1,26 +1,25 @@
 ---
 name: TestSuite
 description: Tests for verifying something
+principals:
+  - &john
+    id: john
+    policyVersion: '20210210'
+    roles:
+      - employee
+    attr:
+      department: marketing
+      geography: GB
+      team: design
 tests:
   - name: Alicia tries to view her own private album
```

Comment:
The test suite seems to follow the "HR application" policy example, but the test is named "Alicia tries to view her own private album". If I am not mistaken, can we align the test and the policies in `internal/test/testdata/store` with the playground "HR application"?

Reply:
Agree on changing the inconsistent test names. That is an omission on my part.

I think that the HR application in the playground has deviated quite a bit from the test policies in `internal/test/testdata/store`.  If we try to sync them, a lot of tests will fail because those "store" policies are used in many places. Since there's no real link between the playground and these tests, let's not bother with that.

Reply:
I see.

What do you think of having root-level principals and resources and then referring?
```yaml
principals:
  - &john
    id: john
    policyVersion: '20210210'
    roles:
      - employee
    attr:
      department: marketing
      geography: GB
      team: design
```
and then a policy can refer to the principal with YAML anchor
```yaml
principal: *john
```

Reply:
I remember we discussed this. I submitted a draft PR for transparency and to collect comments if there are any. I think an RFC would be overkill here.
I will be following this approach to add root level resources, then I will add more tests and will update documentation showcasing this.

Reply:
I commented below about prototyping a realistic test suite. Let's see what's the most user-friendly way to model a test matrix is. Also worth remembering is that there will be more than one policy to test and the user might want to have a common set of principals, resources etc. to be shared between all test suites.